scripts/Tools/preview_namelists

```diff
@@ -15,50 +15,38 @@ from CIME.utils             import expect
 import argparse, doctest
 
 ###############################################################################
-def parse_command_line(args, description):
+def parse_command_line(args):
 ###############################################################################
-    parser = argparse.ArgumentParser(
-        usage="""\n%s [--debug]
-OR
-%s --verbose
-OR
-%s --help
-OR
-%s --test
-
-\033[1mEXAMPLES:\033[0m
-    \033[1;32m# Run \033[0m
-    > %s
-""" % ((os.path.basename(args[0]), ) * 5),
-
-description=description,
-
-formatter_class=argparse.ArgumentDefaultsHelpFormatter
-)
-
+    cime_model = CIME.utils.get_model()
+    parser = argparse.ArgumentParser()
     CIME.utils.setup_standard_logging_options(parser)
 
     parser.add_argument("caseroot", nargs="?", default=os.getcwd(),
                         help="Case directory to build")
+    parser.add_argument('--component',
+                        help="Specify component's namelist to build.")
+    parser.add_argument('--test', action='store_true',
+                        help="Run preview_namelist in test mode.")
 
-    args = parser.parse_args(args[1:])
+    args = parser.parse_args()
 
     CIME.utils.handle_standard_logging_options(args)
 
-    return args.caseroot
+    return args
 
 ###############################################################################
 def _main_func(description):
 ###############################################################################
-    if ("--test" in sys.argv):
+    args = parse_command_line(sys.argv)
```

Comment:
I like the args being unpacked because then I only need to add an argument and don't have to then change the return and the function call itself. Much cleaner and simpler.